config/paths.js

```diff
@@ -1,4 +1,10 @@
 import { SCOPE_TYPE } from 'utils/constants';
+import { Stores } from 'models/stores';
+import { StoreProducts } from 'models/storeProducts';
+import { Suppliers } from 'models/suppliers';
+import { SupplierProducts } from 'models/supplierProducts';
+import config from 'config';
+import { authMiddlewareFunc } from 'utils/apiUtils';
```

Comment:
please create a local utils function in the auth middlware for this

Reply:
please also give it a function name that expresses what this action this function performs